tests/main.test.ts

```diff
@@ -1718,6 +1737,32 @@ describe('zip-it-and-ship-it', () => {
     expect(mockSource).toBe(unzippedBinaryContents)
   })
 
+  test('Builds Go functions from an internal-functions dir with a configured displayName', async () => {
+    vi.mocked(shellUtils.runCommand).mockImplementation(async (...args) => {
+      await writeFile(args[1][2], '')
+
+      return {} as any
+    })
+
+    const fixtureName = 'go-source-internal'
+    const { files } = await zipFixture(fixtureName, {
+      length: 2,
+      fixtureDir: join(FIXTURES_DIR, FUNCTIONS_INTERNAL_DIR),
```

Comment:
can we please move the fixtures around and instead of having `tests/fixtures/.netlify/internal-functions/node_internal` do `tests/fixtures/node_internal/.netlify/internal-functions/`

Reply:
✅ !